src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.HandleValue.cs

```diff
@@ -1,24 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Diagnostics;
-using System.Runtime.CompilerServices;
-
-namespace System.Text.Json.Serialization
-{
-    public static partial class JsonSerializer
-    {
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static bool WriteValue(
```

Comment:
I removed this since it was no longer necessary. Please verify it doesn't hurt your readability (especially when mixed with other method calls within the serializer switch statement), @steveharter

Reply:
That's fine for now. If\when we need more code we can add it back.